src/main/java/xfacthd/framedblocks/common/compat/jei/ClickableStack.java

```diff
@@ -1,18 +1,32 @@
 package xfacthd.framedblocks.common.compat.jei;
 
+import mezz.jei.api.ingredients.IIngredientType;
 import mezz.jei.api.ingredients.ITypedIngredient;
 import mezz.jei.api.runtime.IClickableIngredient;
 import net.minecraft.client.renderer.Rect2i;
 import net.minecraft.world.item.ItemStack;
 
 record ClickableStack(ITypedIngredient<ItemStack> stack, Rect2i area) implements IClickableIngredient<ItemStack>
 {
-    @Override
+    @SuppressWarnings("removal")
+	@Override
```

Comment:
No tabs between my sacred 4 spaces (there's a few more places with tabs and probably some I've not even noticed yet) 😅